src/backend/main.ts

```diff
@@ -1161,228 +1182,231 @@ ipcMain.handle('isClientUpdating', async () => {
 
 ipcMain.on('restartClient', () => autoUpdater.quitAndInstall())
 
-// get pid/tid on launch and inject
-ipcMain.handle(
```

Comment:
i basically moved all of this logic out into `handleGameLaunch` so that we could store the promise in `allLaunchHandlerPromises` to await on app quit